types/eslint/index.d.ts

```diff
@@ -929,6 +929,16 @@ export namespace Linter {
 
     // https://eslint.org/docs/developer-guide/working-with-plugins#processors-in-plugins
     interface Processor<T extends string | ProcessorFile = string | ProcessorFile> {
+        /** @deprecated Use `meta.name` instead. */
+        name?: string | undefined;
+
+        /** @deprecated Use `meta.version` instead. */
+        version?: string | undefined;
+
```

Comment:
```suggestion
```

I don't think they are available for parsers and processors?

Reply:
You beat me to the punch, see comment below 😄